CHANGELOG.md

```diff
@@ -1,5 +1,109 @@
 # Changelog
 
+## v1.17.0
+
+- ### Features
+
+  - **feat: capture panics and generate error report - [nilslice], [pull/1888]**
+
+    This PR adds support for wrangler error reporting. Currently, all panics are caught and a report is generated, then written to disk. `wrangler report` or `wrangler report --log <file-name.log>` will upload the corresponding error report to Cloudflare
+
+    [nilslice]: https://github.com/nilslice
+    [pull/1888]: https://github.com/cloudflare/wrangler/pull/1888
+
+- ### Fixes
+
+  - **fix: clarify error messages around durable objects beta - [nilslice], [pull/1921]**
+
+    Disambiguates error message described in #1859, adds more clarity around other places where Durable Object usage in beta may conflict with `wrangler` functionality.
+
+    [nilslice]: https://github.com/nilslice
+    [pull/1921]: https://github.com/cloudflare/wrangler/pull/1921
+
+  - **Fix filtering by extension for js assets - [rubenamorim], [pull/1722]**
+
+    Fixes #1719
```

Comment:
could you make this a link to the issue 👀

Reply:
ah, yes - why does Github convert this syntax everywhere(?) else into links... but not here. good catch!